.github/dependabot.yml

```diff
@@ -0,0 +1,9 @@
+version: 2
+updates:
+  - package-ecosystem: "pip"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+    ignore:
```

Comment:
We should also ignore flake8-tabs - the newer versions are incompatible with latest flake8 (https://gitlab.com/ntninja/flake8-tabs/-/issues/9) and they cause errors for almost every line (https://gitlab.com/ntninja/flake8-tabs/-/issues/8)